manifests/p/PvPLounge/PvPLounge/0.3.0/PvPLounge.PvPLounge.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: PvPLounge.PvPLounge
+PackageVersion: 0.3.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://assets.pvplounge.com/launcher/PvPLounge%20Launcher%20Setup%200.3.0.exe
+  InstallerSha256: df7039a8564a8b72a755b1f22c3e2ed3360363100a38ae241d21b6943c50563f

```

Comment:
```suggestion
  InstallerSha256: df7039a8564a8b72a755b1f22c3e2ed3360363100a38ae241d21b6943c50563f
  Scope: machine
```

Reply:
`C:\Program Files\PvPLounge Launcher`

Reply:
is it not the same?

Reply:
GitHub will show you two changes (InstallerSha256 & Scope) ; only one of them (Scope) will be changed.